t/default-machine-file.t

```diff
@@ -0,0 +1,30 @@
+use feature 'unicode_strings';
+use strict;
+use Setup;
+
+my %ctx = test_init();
```

Comment:
move it here, with

```
my %ctx = test_init(
  nix_config => q|
system-features = test-system-feature
|
);
```

(as documented in https://github.com/regnat/hydra/blob/abff212d0631726502ce8a025e5ac5e4493828aa/t/lib/Setup.pm#L22)